solr/core/src/java/org/apache/solr/security/JWTAuthPlugin.java

```diff
@@ -199,6 +237,28 @@ public void init(Map<String, Object> pluginConfig) {
     lastInitTime = Instant.now();
   }
 
+  /**
+   * Tries for find X509 certificates in the input stream in DER or PEM format.
+   * Supports multiple certs in same stream if multiple PEM certs are concatenated.
+   * @param trustedCertsStream input stream with the contents of either PEM (plaintext) or DER (binary) certs
+   * @return collection of found certificates, else throws exception
+   */
+  static Collection<X509Certificate> parsePemToX509Certs(InputStream trustedCertsStream) {
```

Comment:
Maybe this belongs in CryptoKeys? Or some other util class?

Reply:
Good idea - moving to CryptoKeys and generalizing a bit